SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/AccountsView.kt

```diff
@@ -0,0 +1,541 @@
+                )
+                Text(
+                    text = balance.accountBalanceInFiatFormatted,
+                    modifier = Modifier.align(Alignment.End),
+                    fontFamily = robotoFont,
+                    fontWeight = FontWeight.Normal,
+                    fontSize = customStyles.itemsCodeTextSize,
+                    color = customStyles.itemsCodeTextColor
+                )
+            }
+
+        }
+    }
+}
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun AccountTrades(
+    listPricesViewModel: ListPricesViewModel?,
+    accountsViewModel: AccountsViewModel?
+) {
+    Column() {
+        Text(
+            text = "Trades",
+            modifier = Modifier,
+            textAlign = TextAlign.Center,
+            fontFamily = robotoFont,
+            fontWeight = FontWeight.Normal,
+            fontSize = 24.sp,
+            color = Color.Black
+        )
+        LazyColumn(
+            modifier = Modifier
+                .padding(top = 15.dp)
+        ) {
+            stickyHeader {
+                AccountTradesHeaderItem()
+            }
+            itemsIndexed(items = accountsViewModel?.trades ?: listOf()) { index, item ->
+                AccountTradesItem(
+                    trade = item,
+                    index = index,
+                    listPricesViewModel = listPricesViewModel,
+                    accountsViewModel = accountsViewModel,
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun AccountTradesHeaderItem(
+    styles: AccountsViewStyles = AccountsViewStyles()
+) {
+
+    val priceColor = if (styles.headerTextColor != Color(R.color.list_prices_asset_component_header_color)) {
+        styles.headerTextColor
+    } else {
+        Color.Black
+    }
+
+    Surface(color = Color.Transparent) {
+
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier.padding(bottom = 16.dp)
+        ) {
+
+            Text(
+                text = "Data",
+                fontFamily = robotoFont,
+                fontWeight = FontWeight.Bold,
+                fontSize = styles.headerTextSize,
+                color = priceColor
+            )
+            Text(
+                text = "Amount",
+                modifier = Modifier
+                    .padding(end = 0.dp)
+                    .weight(1f),
+                textAlign = TextAlign.End,
+                fontFamily = robotoFont,
+                fontWeight = FontWeight.Bold,
+                fontSize = styles.headerTextSize,
+                color = priceColor
+            )
+        }
+    }
+}
+
+@Composable
+fun AccountTradesItem(trade: TradeBankModel,
+                       index: Int,
+                       listPricesViewModel: ListPricesViewModel?,
+                       accountsViewModel: AccountsViewModel?,
+                       customStyles: AccountsViewStyles = AccountsViewStyles()
```

Comment:
Margin looks a little bit off... don't we have a linter rule for this?

Reply:
Fixed